crates/biome_lsp/src/capabilities.rs

```diff
@@ -64,7 +65,13 @@ pub(crate) fn server_capabilities(capabilities: &ClientCapabilities) -> ServerCa
         document_formatting_provider: supports_formatter_dynamic_registration,
         document_range_formatting_provider: supports_range_formatter_dynamic_registration,
         document_on_type_formatting_provider: supports_on_type_formatter_dynamic_registration,
-        code_action_provider: Some(CodeActionProviderCapability::Simple(true)),
+        code_action_provider: Some(CodeActionProviderCapability::Options(CodeActionOptions {
+            code_action_kinds: Some(vec![
+                CodeActionKind::SOURCE_FIX_ALL,
+                CodeActionKind::new("source.fixAll.biome"),
```

Comment:
I believe that registering `source.fixAll.biome` should be enough since clients are supposed to do a prefix matching  so for example if user specifies `source.fixAll` to run on save, it should still match more specific `source.fixAll.biome`.

That said, it probably doesn't hurt to have both.